www/shadowfox-updater/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/SrKomodo/shadowfox-updater 1.7.19 v
+
+categories          www
+platforms           darwin linux
+license             MIT
+maintainers         {@ylluminarious orbitalimpact.com:georgedp} openmaintainer
+description         An auto-updater for ShadowFox (a dark theme for Firefox).
+
+long_description    This is a cross-platform installer/uninstaller/updater for ShadowFox, \
+                    which is a universal dark theme for Firefox. You can visit the main ShadowFox \
+                    website at https://overdodactyl.github.io/ShadowFox
+
+checksums           shadowfox-updater-1.7.19.tar.gz \
```

Comment:
The version number should not be hardcoded here. Use `${version}` so that updates are easier.

Reply:
@mf2k Ok, done.